lib/bloc/account/account_bloc.dart

```diff
@@ -186,11 +204,20 @@ class AccountBloc extends Cubit<AccountState> with HydratedMixin {
   }) async {
     _log.v("lnurlPay amount: $amount, comment: '$comment', reqData: $reqData");
     try {
-      return _breezLib.lnurlPay(
+      final lnurlPayResult = await _breezLib.lnurlPay(
         userAmountSat: amount,
         reqData: reqData,
         comment: comment,
       );
+      _paymentResultStreamController.add(
+        PaymentResult(
```

Comment:
Doesn't it make more sense to handle the success action at the caller of lnurlPay and leave the PaymentResult stream for pure lightning success/failed payments?
This way we won't need to differentiate between these in the UI handler.

Reply:
Yes, I was thinking of making that distinction next.

Reply:
Done. https://github.com/breez/c-breez/pull/444/commits/f6586ea67b0abd18c5e9cda48e91b655bd5086e5 https://github.com/breez/c-breez/pull/444/commits/f74fddd178ac95bd3ab66303df15f63ab1afdd80